fetch_data.py

```diff
@@ -364,19 +364,20 @@ def parse_meta_xml_str(meta_xml_str):
     return dim_encodings
 
 
-def get_xml_str_between_tags(meta_xml_str, tag):
-    """Get string encapsulated by a known XML tag.
+def get_xml_str_between_tags(meta_xml_str, tag, occurrence=1):
```

Comment:
Not saying do this now, but we probably want to just use XPath to extract these expressions now that there's a bit more complexity.  Either adapt this or make a ticket to do so.

Reply:
Hey Bruce I'll ticket this. The way this is being done now is certainly a bit hacky - but I think I'll need a bit of research to find the `lxml` or equivalent fix. Related: I'd propose that we move this function and ones like it to a `describe_data` module. We have a few pieces now that just describe the data and unwrap the coverage encodings - but don't actually fetch any data. I think having them in a common module would help us de-duplicate code because we'll be able to see spots to make those calls for flexible + general.

Reply:
As to the linting - I have my editor set to format Python files with Black upon save. So I'm not sure if they hadn't been formatted in a while or what...that seems weird.